rtv/terminal.py

```diff
@@ -405,7 +473,8 @@ def open_editor(self, data=''):
             # in order to read the changes made by some editors (gedit). w+
             # mode does not work!
             with codecs.open(fp.name, 'r', 'utf-8') as fp2:
-                text = ''.join(line for line in fp2 if not line.startswith('#'))
+                text = ''.join(
```

Comment:
If you look in the *.pylintrc* file, I set the line limit to 100 in order to be a bit more lenient in cases like this where it breaks 79 by a character or two. IMO it's cleaner + more readable to keep it on a single line